api/nodes/serializers.py

```diff
@@ -646,10 +650,20 @@ def create(self, validated_data):
         tag_instances = []
         affiliated_institutions = None
         region_id = None
+        license_details = None
         if 'affiliated_institutions' in validated_data:
             affiliated_institutions = validated_data.pop('affiliated_institutions')
         if 'region_id' in validated_data:
             region_id = validated_data.pop('region_id')
+        if 'license_type' in validated_data or 'license' in validated_data:
+            try:
+                license_details = get_license_details(None, validated_data)
+            except ValidationError as e:
+                raise InvalidModelValueError(detail=e.messages[0])
```

Comment:
Drive by: this should be stringified for Py3 `raise InvalidModelValueError(detail=str(e))` or something like that.